CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/svg/SVGGraphBuilder.java

```diff
@@ -217,22 +217,28 @@ public SVGGraphBuilder fromPerspective(final AxisConstants exportPerspective) {
     public SVGData build() throws InterruptedException {
         
         final SVGObject svgGraph = SVGTemplateConstants.LAYOUT.getSVGObject();
-        
         try{
-            preBuild();
-        } catch (final IllegalArgumentException ex){
-            LOGGER.log(Level.SEVERE, ex.getLocalizedMessage(), ex);
-            return null;
+            try{
```

Comment:
can you add a space after "try" (I notice a similar case above that may have been missed previously)